modules/rosa-sharing-vpc-dns-and-roles.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-dns-and-roles_{context}"]
+= Creating your DNS and cluster creation roles
+
+After the *VPC-owning AWS account* user creates a virtual private cloud, subnets, and an IAM role for sharing the VPC resources, the *cluster-creating AWS account* user must reserve an _openshiftapps.com_ DNS domain and create Operator roles to communicate back to the *VPC-owning AWS account*.
+
+.Prerequisites
+
+* You have the ARN for the IAM role that is used to share your VPC.
+
+.Procedure
+
+. The cluster creator reserves an _openshiftapps.com_ a DNS domain with the following command:
++
+[source,terminal]
+----
+$ rosa create dns-domain
+----
++
+The command creates a reserved _openshiftapps.com_ DNS domain.
++
+[source,terminal]
+----
+I: DNS domain '14eo.p1.openshiftapps.com' has been created.
+I: To view all DNS domains, run 'rosa list dns-domains
+----
+. After creating the DNS domain, the cluster creator needs to create an OIDC configuration. See this link:https://access.redhat.com/articles/7031018[Knowledge Centered Solutions article] for more information on the ODIC configuration process. The following command produces the OIDC config ID that you need:
++
+[source,terminal]
+----
+$ rosa create oidc-config
+----
++
+You receive confirmation that the command created an OIDC configuration.
++
+[source,terminal]
+----
+I: To create Operator Roles for this OIDC Configuration, run the following command and remember to replace <user-defined> with a prefix of your choice:
+	rosa create operator-roles --prefix <user-defined> --oidc-config-id 25tu67hq45rto1am3slpf5lq6jargg
+----
+. After the OIDC configuration is created, you create the Operator roles using the following command:
```

Comment:
I don't think you need the "you" in this sentence. The rest of the page doesn't refer to the second person, it just uses imperatives.